lib/socket/socket_c.nit

```diff
@@ -45,18 +45,20 @@ class PollFD
 	# A collection of the events to be watched
 	var events: Array[NativeSocketPollValues]
 
-	init(pid: Int, events: Array[NativeSocketPollValues])
+	# Create a PollFD object from NativePollFD informations
```

Comment:
Maybe add a few lines in the class documentation to explain quickly what is PollFD? Or add a link?